spring-kafka/src/main/java/org/springframework/kafka/core/DefaultKafkaConsumerFactory.java

```diff
@@ -121,18 +121,16 @@ public void setValueDeserializer(Deserializer<V> valueDeserializer) {
 		if (groupId == null && !shouldModifyClientId) {
 			return createKafkaConsumer();
 		}
-		else {
-			Map<String, Object> modifiedConfigs = new HashMap<>(this.configs);
-			if (groupId != null) {
-				modifiedConfigs.put(ConsumerConfig.GROUP_ID_CONFIG, groupId);
-			}
-			if (shouldModifyClientId) {
-				modifiedConfigs.put(ConsumerConfig.CLIENT_ID_CONFIG,
-					(overrideClientIdPrefix ? clientIdPrefix
-							: modifiedConfigs.get(ConsumerConfig.CLIENT_ID_CONFIG)) + clientIdSuffix);
-			}
-			return createKafkaConsumer(modifiedConfigs);
+		Map<String, Object> modifiedConfigs = new HashMap<>(this.configs);
+		if (groupId != null) {
+			modifiedConfigs.put(ConsumerConfig.GROUP_ID_CONFIG, groupId);
 		}
+		if (shouldModifyClientId) {
+			modifiedConfigs.put(ConsumerConfig.CLIENT_ID_CONFIG,
+				(overrideClientIdPrefix ? clientIdPrefix
+						: modifiedConfigs.get(ConsumerConfig.CLIENT_ID_CONFIG)) + clientIdSuffix);
+		}
+		return createKafkaConsumer(modifiedConfigs);
```

Comment:
What is the purpose of this change - seems like just a style preference.

Reply:
@garyrussell I think `if` block with `return` statement makes the following `else` block redundant and removing it reduces a block depth. WDYT?

Reply:
I can't find the rule now, but we really prefer `if...else` style even if there is a `return` operator.